sound/soc/intel/boards/skl_hda_dsp_generic.c

```diff
@@ -27,13 +27,6 @@ static const struct snd_soc_dapm_widget skl_hda_widgets[] = {
 };
 
 static const struct snd_soc_dapm_route skl_hda_map[] = {
-	{ "hifi3", NULL, "iDisp3 Tx"},
-	{ "iDisp3 Tx", NULL, "iDisp3_out"},
-	{ "hifi2", NULL, "iDisp2 Tx"},
-	{ "iDisp2 Tx", NULL, "iDisp2_out"},
-	{ "hifi1", NULL, "iDisp1 Tx"},
-	{ "iDisp1 Tx", NULL, "iDisp1_out"},
-
```

Comment:
These don't seem to have practical purpose.

Reply:
@plbossart @ranj063 I'm starting to lean towards removing skylake/SST support from these shared machine drivers. If there are no working fw+tplg files available, what's the point of having the code maintained in upstream.  But let's discuss this on the list.

Reply:
@plbossart this is now solved in latest patchset, closing